test/test.js

```diff
@@ -72,14 +39,17 @@ function buildUrl(baseUrl, urlParams, pathname) {
 
 describe('Sitespeed Budgets', function() {
     // Need to bump up time for the Sitespeed tests
-    this.timeout(60000);
+    this.timeout(60000 * 2);
```

Comment:
@kakazuo I suppose I'm the one that hard-coded 60000 to start with, but could we possibly make this a constant? And  maybe even configurable?